src/framework/components/light/component.js

```diff
@@ -87,6 +87,9 @@ const _lightPropsDefault = [];
  * angle is specified in degrees. Affects spot lights only. Defaults to 40.
  * @property {number} outerConeAngle The angle at which the spotlight cone has faded to nothing.
  * The angle is specified in degrees. Affects spot lights only. Defaults to 45.
+ * @property {number} lightSize Size of the light used for contact hardening shadows. For area lights
+ * acts as a size multiplier with the area light dimensions. For punctual and directional lights
+ * acts as the actual size of the light. Defaults to 10.0.
```

Comment:
Should this perhaps default to 1? Multiplier of 10 for area lights seems strange.

Reply:
Good point.